src/sinks/util/udp.rs

```diff
@@ -171,7 +171,16 @@ impl Sink for UdpSink {
                         error!(message = "send failed", %error);
                         Ok(AsyncSink::NotReady(line))
                     }
-                    Ok(_) => Ok(AsyncSink::Ready),
+                    Ok(sent) => {
+                        if sent != line.len() {
+                            warn!(
+                                "Could not send all data in one UDP packet, data length: {}, sent: {}.",
```

Comment:
Extending the message to convey why is this important would also be a nice addition, something like `; dropping X bytes.`

Reply:
Thanks! Added that.